src/main/java/org/broadinstitute/hellbender/utils/test/SamAssertionUtils.java

```diff
@@ -27,12 +24,19 @@ private static SamReader getReader(final File sam, final ValidationStringency va
      *  @param reference is allowed to be null
      */
     public static void assertSamsEqual(final File sam1, final File sam2, final ValidationStringency validationStringency, final File reference) throws IOException {
-        try (final SamReader reader1 = getReader(sam1, validationStringency, reference);
-             final SamReader reader2 = getReader(sam2, validationStringency, reference)) {
-            final SamComparison comparison = new SamComparison(reader1, reader2);
-            final boolean equal = comparison.areEqual();
-            Assert.assertTrue(equal, "SAM file " + sam1.getPath() + " differs from expected output:" + sam2.getPath());
-        }
+        final String equalStringent = samsEqualStringent(sam1, sam2, validationStringency, reference);
+        Assert.assertNull(equalStringent, "SAM file " + sam1.getPath() + " differs from expected output:" + sam2.getPath() + " " + equalStringent);
```

Comment:
If `sam2` is always the "expected" output, then it should be named something like `expectedSam`, and `sam1` should be `actualSam`